uvtools/tests/test_dspec.py

```diff
@@ -434,6 +434,20 @@ def get_snr(clean, fftax=1, avgax=0, modes=[2, 20]):
     rfft2 = np.fft.ifft2(res2)
     nt.assert_true(np.median(np.abs(rfft2[:15, :23] / rfft[:15, :23])) < 1)
 
+def test_delay_interpolation_matrix():
```

Comment:
This test looks like it tests the functionality (generating gaps in data and making sure we can interpolate over them), but it might be good to be a little more explicit about what you're testing and how. A docstring can help, as well as some more comments describing some of the main steps.

Reply:
I've added a docstring with an explanation.